common.mk

```diff
@@ -93,14 +93,13 @@ test-openshift: tag
 tag: build
 	VERSIONS="$(VERSIONS)" $(script_env) $(tag)
 
-.PHONY: clean
-clean: clean-images $(clean-hook)
+.PHONY: clean clean-images clean-versions
```

Comment:
Add `clean-hook` to PHONY as well so `make clean` does not fail when no hook is defined

Reply:
argh, yes... it was there and I dropped that.  Thanks.

Reply:
done